src/main/scala/io/iohk/ethereum/consensus/ethash/validators/EthashValidators.scala

```diff
@@ -0,0 +1,8 @@
+package io.iohk.ethereum.consensus
+package ethash.validators
+
+import io.iohk.ethereum.consensus.validators.Validators
```

Comment:
we still have `BlockHeaderValidator` in `io.iohk.ethereum.validators` which doesn't seem to be needed anymore. Can we delete it (the whole package)?

Reply:
[Yes](https://github.com/input-output-hk/mantis/pull/410/commits/c9d0062c3f0c25451c8fb9776b88e6c9015d83a7)!